crates/runtime/src/lib.rs

```diff
@@ -1,3 +1,11 @@
+use std::collections::BTreeMap;
+use std::sync::Arc;
+
+use actix::prelude::*;
+use calimero_primitives::context::ContextId;
+use calimero_utils_actix::global_runtime;
+use store::InMemoryStorage;
```

Comment:
move this line to L22 (since it's not an external crate), and if you can do the same for the `use crate::` ones on L11-14, that'll be great

you can remove the `crate::` prefix as well, we're already at the root, it's unnecessary, thanks